infrastructure/main.tf

```diff
@@ -1,3 +1,57 @@
 provider "azurerm" {
   version = "=1.42.0"
 }
+
+module "reform-blob-router-db" {
```

Comment:
is it not a notification db as we already have a blob router db :)

Reply:
oh dang, got distracted and didn't rename this copy/paste. :bear: with me, soz